ibm_catalog.json

```diff
@@ -0,0 +1,184 @@
+            },
+            "configuration": [
+              {
+                "key": "ibmcloud_api_key",
+                "displayname": "API KEY"
+              },
+              {
+                "key": "provider_visibility",
+                "displayname": "Provider Visibility",
+                "options": [
+                  {
+                    "displayname": "private",
+                    "value": "private"
+                  },
+                  {
+                    "displayname": "public",
+                    "value": "public"
+                  },
+                  {
+                    "displayname": "public-and-private",
+                    "value": "public-and-private"
+                  }
+                ]
+              },
+              {
+                "key": "use_existing_resource_group",
+                "displayname": "Use Existing Resource Group",
+                "required": true
+              },
+              {
+                "key": "resource_group_name",
+                "displayname": "Resource Group Name",
+                "required": true
+              },
+              {
+                "key": "prefix",
+                "displayname": "Prefix",
+                "required": true,
+                "description": "The prefix to add to all resources that this solution creates. To not use any prefix value, you can enter the string `__NULL__`."
+              },
+              {
+                "key": "existing_monitoring_crn",
+                "displayname": "Existing Monitoring CRN"
+              },
+              {
+                "key": "region",
+                "displayname": "Region",
+                "options": [
+                  {
+                    "displayname": "Dallas (us-south)",
+                    "value": "us-south"
+                  },
+                  {
+                    "displayname": "Frankfurt (eu-de)",
+                    "value": "eu-de"
+                  },
+                  {
+                    "displayname": "Madrid (eu-es)",
+                    "value": "eu-es"
+                  },
+                  {
+                    "displayname": "Toronto (ca-tor)",
+                    "value": "ca-tor"
+                  }
+                ]
+              },
+              {
+                "key": "scc_workload_protection_instance_name",
+                "displayname": "Instance Name"
+              },
+              {
+                "key": "scc_workload_protection_service_plan",
+                "displayname": "Service Plan",
+                "options": [
+                  {
+                    "displayname": "Free trial",
+                    "value": "free-trial"
+                  },
+                  {
+                    "displayname": "Graduated tier",
+                    "value": "graduated-tier"
+                  }
+                ]
+              },
+              {
+                "key": "scc_workload_protection_instance_tags",
```

Comment:
we should use the this customisation for tags -> https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager/blob/fef71c374266a78a8387c8f72f7c8ebfc6e80e63/ibm_catalog.json#L152-L158